File: ProAgil.API/Models/Evento.cs
namespace ProAgil.API.Models
{
    public class Evento
    {
        public int eventoId { get; set; }
        public string imagemUrl { get; set; }
        public string local { get; set; }
        public string dataDoEvento { get; set; }
        public string tema { get; set; }
        public int quantidadeDePessoas { get; set; }
        public string lote { get; set; }
    }
}

File: ProAgil.App/src/app/_models/Evento.ts
import { Lote } from './Lote';
import { RedeSocial } from './RedeSocial';
import { Palestrante } from './Palestrante';

export class Evento {

    constructor() { }

    id: number;
    imagemUrl: string;
    local: string;
    dataDoEvento: Date;
    tema: string;
    telefone: string;
    email: string;
    quantidadeDePessoas: number;
    lotes: Lote[];
    redesSociais: RedeSocial[];
    palestrantesEventos: Palestrante[];
}
